### PR TITLE
Create a Contact me page

### PR DESCRIPTION
**Pull request by Rizwan-learn**
A page to be added to allow the users to put in their information and send it to me.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -285,6 +285,33 @@ <h1>Skills</h1>
         </div>
     </section> 
     
+    <section class="contact-form">
+        <h1 class="title-contact">Contact Me</h1>
```
Comment (ahmetbozaci) [inline index.html]: - [ ] You have already one h1 tag on the page. The page should include only one [h1 tag](https://www.w3schools.com/tags/tag_hn.asp#:~:text=Note%3A%20Only%20use%20one%20%3Ch1%3E%20per%20page%20%2D%20this%20should%20represent%20the%20main%20heading/subject%20for%20the%20whole%20page.%20Also%2C%20do%20not%20skip%20heading%20levels%20%2D%20start%20with%20%3Ch1%3E%2C%20then%20use%20%3Ch2%3E%2C%20and%20so%20on.). Please change to another hx tag.

File: index.html [hunk 2]
```diff
@@ -285,6 +285,33 @@ <h1>Skills</h1>
...
+        <h2 class="view">If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. 
+            I’d love to help with it 
+        </h2>   
+
```
Comment (ahmetbozaci) [inline index.html]: - [ ] Original design has background-image. Please add it. Also, make sure the color background color is the same.
- [ ] Form is the center of the page in the original design. Please fix it.
image
- [ ] In order to layout the elements on the page you should use Flexbox. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20on%20the%20page%20you%20should%20use%20Flexbox.)

### ORIGINAL
image

File: index.html [hunk 3]
```diff
@@ -285,6 +285,33 @@ <h1>Skills</h1>
...
+            <br>
+            <br>         
+            <input type="text" class="box" placeholder="Enter text here" maxlength="500">
+            <br>
+            <br>
+            <div class="get-in-touch">
+                <button type="button"> Get In Touch </button>
+            </div> 
+
+        </form>
```
Comment (ahmetbozaci) [inline index.html]: - [ ] Make sure placeholders are the same as in the original design. Of course, you will change your name and mail with your name and mail.
- [ ] All input fields should have the `required` attribute. Please add it.
- [ ] Form uses the action and method attribute to integrate with formspree service. Please fix it.
[Requirement.
](https://formspree.io/html)
image
- [ ] Message input's should be `<textarea>`. Please change it

File: index.html [hunk 4]
```diff
@@ -285,6 +285,33 @@ <h1>Skills</h1>
...
+            <br>
+            <input type="email" class="input" placeholder="Email address">
+            <br>
+            <br>         
+            <input type="text" class="box" placeholder="Enter text here" maxlength="500">
+            <br>
+            <br>
+            <div class="get-in-touch">
+                <button type="button"> Get In Touch </button>
+            </div>
```
Comment (ahmetbozaci) [inline index.html]: - [ ] You should implement the button interactions (enable, hover, etc.).
- [ ] The submit button can be either an `input` element with type `submit` or a `button` element with type `submit` or with no `type` attribute. Please fix it.

image

File: styles.css [hunk 5]
```diff
@@ -603,3 +602,64 @@ button:active {
...
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 20px;
+  text-align: center;
+  color: #ebebff;
+  margin-right: 10%;
+  margin-left: 5%;
+  padding-bottom: 10%;
+}
```
Comment (ahmetbozaci) [inline styles.css:637]: ### OPTIONAL
- [ ] For DRY CSS you should combine duplicate properties. For example, you can create a class `.font-family-poppins{ font-family: Poppins}` and use this class in an HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️

### Hello @Rizwan-learn  ✋🏼 
#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.`***

**OPTIONAL**

- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
___

### Cheers and Happy coding! 💻 👏

almost

___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: index.html (hunk 1 above)
Comment (Rizwan-learn) [reply index.html]: H1 changed to h2 and h3

File: index.html (hunk 2 above)
Comment (Rizwan-learn) [reply index.html]: background changed and aligned

File: index.html (hunk 3 above)
Comment (Rizwan-learn) [reply index.html]: textarea added, required added.

File: index.html (hunk 4 above)
Comment (Rizwan-learn) [reply index.html]: button attribute added

File: index.html [hunk 6]
```diff
@@ -285,6 +285,29 @@ <h1>Skills</h1>
...
+            <br>
+            <textarea class="box" placeholder="Enter text here" required maxlength="500"></textarea>
+            <br>
+            
+            <button type="button" class="get-in-touch"> Get In Touch </button>
+             
+        </form>
+
+        <br>
+        <br>
```
Comment (rotimiazeez) [inline index.html]: - [ ] Please note that the form is not submitting when one click on submit.🙄Kindly fix this issue👍

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team👋, 

### Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi, @Rizwan-learn 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**